node_modules/oae-core/topnavigation/js/topnavigation.js

```diff
@@ -112,6 +112,11 @@ define(['jquery', 'oae.core', 'activityadapter'], function($, oae, ActivityAdapt
          * to jump past the top navigation
          */
         var setUpSkipLinks = function() {
+            // Index page has a unique structure.
+            if ($('#index-page-container').length > 0) {
```

Comment:
Rather than referencing the index page in the top navigation widget, can we just add the `oae-page-content` id to the appropriate point in the index page? When doing this, we might want to make that id a bit more generic (e.g. `oae-main-content`)